### ISSUE TITLE
Missing field in MatchParticipant class.

### ISSUE THREAD

**Issue by mastrayer** [stelar7/R4J#39]
hi @stelar7 !

I think missing field in lol match v5 api response.
This is "challenges" field in Match Participant.

```
{
  ...
  "challenges": {
                "12AssistStreakCount": 0,
                "abilityUses": 168,
                "acesBefore15Minutes": 0,
                "alliedJungleMonsterKills": 0,
                "baronTakedowns": 0,
                "blastConeOppositeOpponentCount": 0,
                "bountyGold": 0,
                "buffsStolen": 0,
                "completeSupportQuestInTime": 0,
                "controlWardTimeCoverageInRiverOrEnemyHalf": 0.05215168915369436,
                "controlWardsPlaced": 3,
                "damagePerMinute": 529.0015434423793,
                "damageTakenOnTeamPercentage": 0.22122323997443094,
                "dancedWithRiftHerald": 0,
                "deathsByEnemyChamps": 5,
                "dodgeSkillShotsSmallWindow": 39,
                "doubleAces": 0,
                "dragonTakedowns": 0,
                "earlyLaningPhaseGoldExpAdvantage": -0.06373274326324463,
                "effectiveHealAndShielding": 0,
                "elderDragonKillsWithOpposingSoul": 0,
                "elderDragonMultikills": 0,
                "enemyChampionImmobilizations": 35,
                "enemyJungleMonsterKills": 0,
                "epicMonsterKillsNearEnemyJungler": 0,
                "epicMonsterKillsWithin30SecondsOfSpawn": 0,
                "epicMonsterSteals": 0,
                "epicMonsterStolenWithoutSmite": 0,
                "flawlessAces": 0,
                "fullTeamTakedown": 0,
                "gameLength": 1614.2859992416377,
                "getTakedownsInAllLanesEarlyJungleAsLaner": 0,
                "goldPerMinute": 338.45626803455053,
                "hadAfkTeammate": 0,
                "hadOpenNexus": 0,
                "highestCrowdControlScore": 1,
                "immobilizeAndKillWithAlly": 3,
                "initialBuffCount": 0,
                "initialCrabCount": 0,
                "jungleCsBefore10Minutes": 0,
                "junglerTakedownsNearDamagedEpicMonster": 0,
                "kTurretsDestroyedBeforePlatesFall": 0,
                "kda": 1.6,
                "killAfterHiddenWithAlly": 1,
                "killParticipation": 0.2,
                "killedChampTookFullTeamDamageSurvived": 0,
                "killsNearEnemyTurret": 1,
                "killsOnOtherLanesEarlyJungleAsLaner": 0,
                "killsOnRecentlyHealedByAramPack": 0,
                "killsUnderOwnTurret": 0,
                "killsWithHelpFromEpicMonster": 0,
                "knockEnemyIntoTeamAndKill": 2,
                "landSkillShotsEarlyGame": 3,
                "laneMinionsFirst10Minutes": 68,
                "laningPhaseGoldExpAdvantage": -0.08101105690002441,
                "legendaryCount": 0,
                "lostAnInhibitor": 0,
                "maxCsAdvantageOnLaneOpponent": 5,
                "maxKillDeficit": 0,
                "maxLevelLeadLaneOpponent": 1,
                "moreEnemyJungleThanOpponent": -102.00000002980232,
                "multiKillOneSpell": 0,
                "multiTurretRiftHeraldCount": 0,
                "multikills": 0,
                "multikillsAfterAggressiveFlash": 0,
                "mythicItemUsed": 6662,
                "outerTurretExecutesBefore10Minutes": 0,
                "outnumberedKills": 0,
                "outnumberedNexusKill": 0,
                "perfectDragonSoulsTaken": 0,
                "perfectGame": 0,
                "pickKillWithAlly": 7,
                "poroExplosions": 0,
                "quickCleanse": 0,
                "quickFirstTurret": 0,
                "quickSoloKills": 0,
                "riftHeraldTakedowns": 0,
                "saveAllyFromDeath": 0,
                "scuttleCrabKills": 0,
                "skillshotsDodged": 9,
                "skillshotsHit": 25,
                "snowballsHit": 0,
                "soloBaronKills": 0,
                "soloKills": 0,
                "soloTurretsLategame": 1,
                "stealthWardsPlaced": 3,
                "survivedSingleDigitHpCount": 0,
                "survivedThreeImmobilizesInFight": 6,
                "takedownOnFirstTurret": 0,
                "takedowns": 8,
                "takedownsAfterGainingLevelAdvantage": 0,
                "takedownsBeforeJungleMinionSpawn": 0,
                "takedownsFirst25Minutes": 7,
                "takedownsInAlcove": 0,
                "takedownsInEnemyFountain": 0,
                "teamBaronKills": 0,
                "teamDamagePercentage": 0.2184990967612713,
                "teamElderDragonKills": 0,
                "teamRiftHeraldKills": 0,
                "threeWardsOneSweeperCount": 0,
                "tookLargeDamageSurvived": 0,
                "turretPlatesTaken": 2,
                "turretTakedowns": 1,
                "turretsTakenWithRiftHerald": 0,
                "twentyMinionsIn3SecondsCount": 0,
                "unseenRecalls": 0,
                "visionScoreAdvantageLaneOpponent": 1.0371013879776,
                "visionScorePerMinute": 0.7207488146496799,
                "wardTakedowns": 5,
                "wardTakedownsBefore20M": 2,
                "wardsGuarded": 0
            }
  ...
}
```

Could you implement this field?
Thanks.

**Comment by stelar7** [comment]
Fixed in 2.1.7

**Comment by mastrayer** [comment]
Thanks!
